packages/scss/src/components/dialog/component.scss

```diff
@@ -88,6 +88,12 @@
 			}
 		}
 
+		.dialog-inside-header-actionOptional {
+			&:empty {
+				display: none;
```

Comment:
Visiblement oui ^^

Reply:
It works great! Perhaps a pattern to share, I have the feeling that it's not very instinctive.